clients/cobol-lsp-vscode-extension/src/task/CustomBuildTaskTerminal.ts

```diff
@@ -0,0 +1,167 @@
+        const sysPrintContent = await jesApi.getSpoolContentById(submitJob.jobname, submitJob.jobid, sysPrintSpool[0].id);
+
+        this.writeEmitter.fire(`Analysing output...${os.EOL}`);
+
+        this.writeEmitter.fire(sysPrintContent.split("\n")
+            .filter(str => str.match(this.ERROR_REGEX))
+            .map(str => {
+                const match = str.match(this.ERROR_REGEX);
+                return `\x1b[4m\x1b[33m${this.documentText.uri.fsPath}:${match[1]}:7\x1b[24m\x1b[39m -72 - ${match[2]}: ${match[3]}`;
+            })
+            .join(os.EOL));
+        this.storeSysPrint(submitJob, sysPrintContent, loadedProfile.profile.host.split(".")[0]);
+        }
+    }
+
+    private storeSysPrint(submitJob: any, sysPrintContent: any, hostname: string) {
+        if (this.definition.saveListing) {
+            const defaultLocation = path.join(`${vscode.workspace.workspaceFolders[0].uri.fsPath}`, ".c4z", "listings", `${hostname}`);
+            const fileLocation = this.definition.listingLocation ? `${this.definition.listingLocation}` : defaultLocation;
+            if (!fs.existsSync(fileLocation)) {
+                fs.mkdirSync(fileLocation, { recursive: true });
+            }
+            fs.writeFileSync(`${fileLocation}${path.sep}${submitJob.jobid}`, sysPrintContent);
+            this.writeEmitter.fire(os.EOL);
+            this.writeEmitter.fire(`SYSPRINT at \x1b[4m\x1b[33m${fileLocation}${path.sep}${submitJob.jobid}\x1b[24m\x1b[39m${os.EOL}`);
+        }
+    }
+
+    private getCompileJobJCL() {
+        let jclToSubmit: string;
+        const jobCard = (this.definition.jobCard as string[]).join(os.EOL);
+        const compilerOptions = this.getCompilerOptions();
+        const jclHeader = compilerOptions.length > 0 ? `${COMPILE_JCL_HEADER},${os.EOL}${compilerOptions}` : COMPILE_JCL_HEADER;
+        const steplibs = this.definition.steplib && this.definition.steplib.length > 0 ? this.generateLibs("STEPLIB", this.definition.steplib) : "//*";
+        const syslibs = this.definition.syslib && this.definition.syslib.length > 0 ? this.generateLibs("SYSLIB", this.definition.syslib) : "//*";
+        jclToSubmit = `${jobCard}${os.EOL}${jclHeader}${os.EOL}${steplibs}${os.EOL}${COMPILE_JCL_MIDDLE}${os.EOL}${syslibs}${os.EOL}${COMPILE_JCL_TAIL}${os.EOL}${this.documentText.getText()}`;
+        return jclToSubmit;
+    }
+
+    private getCompilerOptions(): string {
+        if (typeof this.definition.compilerOptions !== "undefined" && this.definition.compilerOptions.length > 0)
+        return "// PARM=(" + this.formatCompilerOptions(this.definition.compilerOptions.join(","), 62) + ")";
+        return "";
+    }
+
+    private generateLibs(libName: string, params: string[]) {
+        return params.map((str, i) => {
+            if (i === 0) {
+                return `//${libName}  DD  DISP=SHR,DSN=${str}`;
+            }
+            return `//         DD  DISP=SHR,DSN=${str}`;
```

Comment:
make sure that the line has at most 71 characters when `libname` has 8 chars and `str` 44